eta-company-calendar-frontend/src/app/header/components/header.component.ts

```diff
@@ -81,8 +81,17 @@ import { ProfilViewDialog } from '../../shared/modals/profil-view-dialog.compone
           >{{'header.userManagement' | translate}}</a>
     </span>
       <div class="header2">
-      <a class="username">{{userName}}</a>
-        <button class="translate-button" (click)="onLanguageChange()">{{'header.button' | translate}}</button>
+      <mat-icon style="vertical-align: middle !important;" color="primary"
+        matTooltip="{{ 'header.activeUserIcon' | translate }}" matTooltipPosition="before">
+         account_circle
+      </mat-icon>
+      <a class="username">  {{userName}}</a>
+        <button matTooltip="{{ 'header.languageChange' | translate }}{{'header.button' | translate }}"
+        matTooltipPosition="before" class="translate-button" (click)="onLanguageChange()">
+          <mat-icon style="vertical-align: middle !important;" color="primary">
```

Comment:
oszt ez itt mi? nem elég az inline css, még !important is? :D

Reply:
Átírtam!